cli/tests/data/test_switch_port_settings_show.stdout

```diff
@@ -1,7 +1,7 @@
 switch0/qsfp0
 =============
-Autoneg  Fec   Speed
-false    None  Speed100G
+Autoneg  Fec     Speed
+false    <auto>  speed100_g
 
 Address          Lot            VLAN
 169.254.10.2/30  initial-infra  None
```

Comment:
@rcgoodfellow I changed this to have what I thought was more reasonable output (i.e. not using Debug). If you don't like it, please let me know.

Reply:
Looks fine to me